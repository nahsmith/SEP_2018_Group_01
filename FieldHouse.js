import React, { Component } from 'react';
import { StyleSheet, ScrollView, View, Alert, Image, Dimensions, TouchableOpacity, Navigator} from 'react-native';
import { Container, Header, Title, Content, Footer, FooterTab, Button, Left, Right, Body, Icon, Text } from 'native-base';
import ScrollToTop from 'react-native-scroll-to-top';
import { StackNavigator, NavigationActions } from 'react-navigation';

const deviceWidth = Dimensions.get('window').width;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#F5FCFF',
  },
  welcome: {
    fontSize: 20,
    textAlign: 'center',
    margin: 10,
  },
});

class FieldHouse extends Component {
    render() {
        const { state, navigate } = this.props.navigation;
        const resetAction = NavigationActions.reset({
            index: 0,
            actions: [NavigationActions.navigate({ routeName: 'HomeScreen' })],
          });
        return (
    <Container>
            <Header style={{backgroundColor: 'gold'}}>
	          <Left>
                <Button transparent dark onPress={() => this.props.navigation.goBack()}>
	          	    <Icon name='arrow-dropleft' />
	            </Button>
	          </Left>
	          <Body>
	          	 <Title>Field House</Title>
	          </Body>
	          <Right>
	          <Button transparent dark onPress={() => Alert.alert('You Pressed','Forums!')}>
	          	<Icon name='chatbubbles' />
	          </Button>
	          <Button transparent dark onPress={() => this.props.navigation.dispatch(resetAction)}>
	          	<Icon name='home' />
	          </Button>
	          </Right>
	        </Header>
	        <Content>
	          <ScrollView scrollsToTop={true} ref={(ref) => this.myScroll = ref}>
	          <View style={styles.container}>
            	    <Image style={{width: deviceWidth}} resizeMode='cover' source={require('./images/FieldHouse.jpg')}/>
            	    <Text style={{textAlign: 'center', fontSize: 30}}>{'\nField House'}</Text>
            	    <Text style={{textAlign: 'center', fontSize: 20}}>{'\nHours'}</Text>
            	    <Text style={{textAlign: 'center', fontSize: 15}}>{'\nMonday: 5AM - 7PM'}</Text>
            	    <Text style={{textAlign: 'center', fontSize: 15}}>{'Tuesday: 5:30AM - 8PM'}</Text>
                    <Button style={{alignItems: 'center'}} transparent dark onPress={() => navigate("RecCenter", {screen: "Rec Center"})}>
                        <Text style={{textAlign: 'center', fontSize: 30, color: 'blue'}}>{'\n\n\n\n\n\n     Take Me To Rec Center'}</Text>
                    </Button>
                    <Text style={{textAlign: 'center', fontSize: 15}}>{'\n\n\n\n'}</Text>
	         	</View>
        </ScrollView>
	        </Content>
	        {/*
	        <Footer>
	          <FooterTab>
	            <Button full>
	              <Text>Footer</Text>
	            </Button>
	          </FooterTab>
	        </Footer>
	        */}
      </Container>
        );
    }
}

FieldHouse.navigationOptions = {
        title: 'Field House',
        header: false,
};

export default FieldHouse
