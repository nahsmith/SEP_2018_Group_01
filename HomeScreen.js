import React, { Component } from 'react';
import { StyleSheet, ScrollView, View, Alert, Image, Dimensions, TouchableOpacity, TouchableHighlight, Navigator} from 'react-native';
import { Container, Header, Title, Content, Footer, FooterTab, Button, Left, Right, Body, Icon, Text } from 'native-base';
import ScrollToTop from 'react-native-scroll-to-top';
import { StackNavigator, NavigationActions } from 'react-navigation';
import { Ionicons } from '@expo/vector-icons';

const deviceWidth = Dimensions.get('window').width;

//var firebase = require("firebase");
import * as firebase from "firebase";
//import { database } from 'firebase';

// Initialize Firebase
var src="https://www.gstatic.com/firebasejs/4.9.1/firebase.js";
var config = {
 apiKey: "AIzaSyD5Ha8fSPEBkjqMchNVy8zU6vB-WvcGYTM",
 authDomain: "herkyles-85e8e.firebaseapp.com",
 databaseURL: "https://herkyles-85e8e.firebaseio.com",
 projectId: "herkyles-85e8e",
 storageBucket: "herkyles-85e8e.appspot.com",
 messagingSenderId: "812934741247"
};
firebase.initializeApp(config);
var databaseRef = firebase.database();
var imageRef = firebase.storage();
var test1;
var test2;
var arrayTest = [];
var imageArray = [];

export default class HomeScreen extends Component {
  
    static navigationOptions = {
        title: "Home",
        header: false,
    }

    componentDidMount() {
        
    }

    constructor(props) {
        super(props);
        console.log("constructor")
        this.state={
            gym1: "",
            gym2: "",
            image1: "",
            image2: "",
        }
        var that = this;
        var mainRef = firebase.database().ref("facilities")
        //console.log(mainRef.once("value")); 
        //var recCenterRef = databaseRef.ref("facilities/reccenter");
        //var fieldHouseRef = databaseRef.ref("facilities/fieldhouse")
        var mainImageRef = imageRef.ref("mainImages/CampusRec.jpg");
        mainImageRef.getDownloadURL().then(function(url) {
          console.log(url);
          imageArray[0] = url;
          that.setState({
            image1 : imageArray[0],
          })
        });
        
        mainRef.once("value").then(function(dataSnapshot) {
          console.log(dataSnapshot);
          var i = 0;
          dataSnapshot.forEach(function(testingSnap){
            arrayTest[i] = testingSnap.child("name").val();
            //console.log(arrayTest[i]);
            i++;
          })
          that.setState({
            gym1 : arrayTest[0],
            gym2 : arrayTest[1],
          })
        })

        /*mainImageRef.getDownloadURL().then(function(url) {
          console.log(url);
        });
        */

        /*mainImageRef.once("value").then(function(imageSnapshot) {
          console.log(imageSnapshot);
          var i = 0;
          imageSnapshot.forEach(function(testingSnap){
            imageArray[i] = testingSnap.child.val();
            //console.log(arrayTest[i]);
            i++;
          })
          that.setState({
            image1 : imageArray[0],
            //gym2 : arrayTest[1],
          })
        })
        */

       
    }
  
  render() {
    const { navigate } = this.props.navigation;
    const resetAction = NavigationActions.reset({
        index: 0,
        actions: [NavigationActions.navigate({ routeName: 'HomeScreen' })],
      });
    return (
    	<Container>
            <Header style={{backgroundColor: 'gold'}}>
	          <Left>
	          </Left>
	          <Body>
	          	 <Title>Herkyles</Title>
	          </Body>
	          <Right>
	          <Button transparent dark onPress={() => Alert.alert('You Pressed','Forums!')}>
	          	<Icon name='chatbubbles' />
	          </Button>
	          </Right>
	        </Header>
	        <Content>
	          <ScrollView scrollsToTop={true} ref={(ref) => this.myScroll = ref}>
	          <View style={styles.container}>
	          <TouchableOpacity activeOpacity={ 0.75 } style={ styles.button } onPress={() => navigate("RecCenter", {screen: "Rec Center"})}>
            	      <Image style={{width: deviceWidth}} resizeMode='cover' source = {{uri: this.state.image1.toString}}/>
            	      <Text style={{textAlign: 'center', fontSize: 30}}>{'\n' + this.state.gym1}</Text>
            	      <Text style={{textAlign: 'center', fontSize: 20}}>{'\nHours'}</Text>
            	      <Text style={{textAlign: 'center', fontSize: 15}}>{'\nWeekday: 5AM - 7PM'}</Text>
            	      <Text style={{textAlign: 'center', fontSize: 15}}>{'Weekend: 5:30AM - 8PM'}</Text>
			 </TouchableOpacity>
	         	</View>
	         	<View style={styles.container}>
	          <TouchableOpacity activeOpacity={ 0.75 } style={ styles.button } onPress={() => navigate("FieldHouse", {screen: "Field House"})}>
            	      <Image style={{width: deviceWidth}} resizeMode='cover' source={require('./images/FieldHouse.jpg')}/>
            	      <Text style={{textAlign: 'center', fontSize: 30}}>{'\n' + this.state.gym2}</Text>
            	      <Text style={{textAlign: 'center', fontSize: 20}}>{'\nHours'}</Text>
            	      <Text style={{textAlign: 'center', fontSize: 15}}>{'\nWeekday: 5AM - 7PM'}</Text>
            	      <Text style={{textAlign: 'center', fontSize: 15}}>{'Weekend: 5:30AM - 8PM'}</Text>
			 </TouchableOpacity>
	         	</View>
	         	<View style={styles.container}>
	          <TouchableOpacity activeOpacity={ 0.75 } style={ styles.button } onPress={() => Alert.alert('You Pressed','#3!',[{text: 'OK', onPress: () => console.log('OK Pressed')}])}>
            	      <Image style={{width: deviceWidth}} resizeMode='cover' source={require('./images/FitnessEast.jpg')}/>
            	      <Text style={{textAlign: 'center', fontSize: 30}}>{'\nFitness East Recreational Facilty'}</Text>
            	      <Text style={{textAlign: 'center', fontSize: 20}}>{'\nHours'}</Text>
            	      <Text style={{textAlign: 'center', fontSize: 15}}>{'\nWeekday: 5AM - 7PM'}</Text>
            	      <Text style={{textAlign: 'center', fontSize: 15}}>{'Weekend: 5:30AM - 8PM'}</Text>
			 </TouchableOpacity>
	         	</View>
        </ScrollView>
	        </Content>
      </Container>
    );
  }
}
function refresh(test1){
  console.log("HERE: " + test1)
}
function testing(test1){
  this.setState({gym1: test1}, refresh(test1))
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
  },
  bigred: {
    color: 'red',
    fontWeight: 'bold',
    fontSize: 50,
  },
  buttonContainer: {
    margin: 20,
    marginTop: 100
  },
  alternativeLayoutButtonContainer: {
    margin: 20,
    flexDirection: 'row',
    justifyContent: 'space-between'
  },
  button: {
  	flex: 1,
  	flexDirection: 'column',
  	padding: 0,
  	justifyContent: 'center',
    marginBottom: 20,
    shadowColor: '#303838',
    shadowOffset: { width: 0, height: 5 },
    shadowRadius: 10,
    shadowOpacity: 0.35,
  },
  title:{
  	
  }
});
