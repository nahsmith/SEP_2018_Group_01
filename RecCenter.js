import React, { Component } from 'react';
import { StyleSheet, ScrollView, View, Alert, Image, Dimensions, TouchableOpacity, Navigator} from 'react-native';
import { Container, Header, Title, Content, Footer, FooterTab, Button, Left, Right, Body, Icon, Text } from 'native-base';
import ScrollToTop from 'react-native-scroll-to-top';
import { StackNavigator, NavigationActions } from 'react-navigation';

const deviceWidth = Dimensions.get('window').width;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#F5FCFF',
  },
  welcome: {
    fontSize: 20,
    textAlign: 'center',
    margin: 10,
  },
});

class RecCenter extends Component {
    render() {
				const { state, navigate } = this.props.navigation;
				var gymInfo=[];
				var displayInfo=[];
				var i=0;
				state.params.db.forEach(function(data){
						 gymInfo.push(data)
				})
				var workoutAreaName=[]
				gymInfo[0].child("workoutArea").forEach(function(area){
					workoutAreaName[i]=area.child("name");
					i++;
				})
				// console.log(workoutAreaName)

        const resetAction = NavigationActions.reset({
            index: 0,
            actions: [NavigationActions.navigate({ routeName: 'HomeScreen' })],
					});
				for(i=0; i <workoutAreaName.length;i++){
					displayInfo.push(<Text style={{textAlign: 'center', fontSize: 15}}>{workoutAreaName[i]}</Text>)
				}
				// console.log(displayInfo);
        return (
    <Container>
            <Header style={{backgroundColor: 'gold'}}>
	          <Left>
                <Button transparent dark onPress={() => this.props.navigation.goBack()}>
	          	    <Icon name='arrow-dropleft' />
	            </Button>
	          </Left>
	          <Body>
	          	 <Title>Rec Center</Title>
	          </Body>
	          <Right>
	          <Button transparent dark onPress={() => Alert.alert('You Pressed','Forums!')}>
	          	<Icon name='chatbubbles' />
	          </Button>
	          <Button transparent dark onPress={() => this.props.navigation.dispatch(resetAction)}>
	          	<Icon name='home' />
	          </Button>
	          </Right>
	        </Header>
	        <Content>
	          <ScrollView scrollsToTop={true} ref={(ref) => this.myScroll = ref}>
	          <View style={styles.container}>
            	    <Image style={{width: deviceWidth}} resizeMode='cover' source={require('./images/CampusRec.jpg')}/>
            	    <Text style={{textAlign: 'center', fontSize: 30}}>{'\n'+gymInfo[0].child("name").val()}</Text>

									<Text style={{textAlign: 'center', fontSize: 20}}>{'\nAddress'}</Text>
            	    <Text style={{textAlign: 'center', fontSize: 15}}>{gymInfo[0].child("address").val()}</Text>

									<Text style={{textAlign: 'center', fontSize: 20}}>{'\nPhone'}</Text>
            	    <Text style={{textAlign: 'center', fontSize: 15}}>{gymInfo[0].child("phone").val()}</Text>

            	    <Text style={{textAlign: 'center', fontSize: 20}}>{'\nHours'}</Text>
            	    <Text style={{textAlign: 'center', fontSize: 15}}>{gymInfo[0].child("hours").child("open").child("weekdays").val()}</Text>
            	    <Text style={{textAlign: 'center', fontSize: 15}}>{gymInfo[0].child("hours").child("open").child("weekends").val()}</Text>
                    {/* <Button style={{alignItems: 'center'}} transparent dark onPress={() => navigate("FieldHouse", {screen: "Field House"})}>
                        <Text style={{textAlign: 'center', fontSize: 30, color: 'blue'}}>{'\n\n\n\n\n\n    Take Me To Field House'}</Text>
                    </Button> */}
                    <Text style={{textAlign: 'center', fontSize: 15}}>{'\n\n\n\n'}</Text>
	         	</View>
        </ScrollView>
	        </Content>
	        {/*
	        <Footer>
	          <FooterTab>
	            <Button full>
	              <Text>Footer</Text>
	            </Button>
	          </FooterTab>
	        </Footer>
	        */}
      </Container>
        );
    }
}

RecCenter.navigationOptions = {
    title: `Rec Center`,
    header: false,
};

export default RecCenter
