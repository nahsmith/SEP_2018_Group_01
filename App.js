/*
	YOU MUST RUN EACH OF THESE DEPENDENCIES IN THE CONSOLE PRIOR TO RUNNING
	
	yarn add react
	yarn add react-native
	yarn add native-base
	yarn add react-native-scroll-to-top
	yarn add react-navigation
*/

import React from 'react';

import { StackNavigator } from 'react-navigation';
import HomeScreen from './components/HomeScreen.js';
import RecCenter from './components/RecCenter.js';
import FieldHouse from './components/FieldHouse.js';

const App = StackNavigator({
    HomeScreen: { screen: HomeScreen},
	RecCenter: { screen: RecCenter},
	FieldHouse: { screen: FieldHouse}
})

export default App;