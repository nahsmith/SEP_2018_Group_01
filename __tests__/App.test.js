import React from 'react';
import App from './../App';
import HomeScreen from './../components/HomeScreen';
import FieldHouse from './../components/FieldHouse';
import RecCenter from './../components/RecCenter';
import renderer from 'react-test-renderer';
import { Container, Header, Title, Content, Footer, FooterTab, Button, Left, Right, Body, Icon, Text } from 'native-base';
import ShallowRenderer from 'react-test-renderer/shallow';
import ReactTestUtils from 'react-dom/test-utils';

afterAll(() => setTimeout(() => process.exit(), 1000));

describe('App.js', () => {
  it('App renders without crashing', () => {
    const rendered = renderer.create(<App />).toJSON();
    expect(rendered).toBeTruthy();
    expect(rendered).toMatchSnapshot();
  })
});