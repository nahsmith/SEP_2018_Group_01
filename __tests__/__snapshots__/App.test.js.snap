// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.js App renders without crashing 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
    ]
  }
>
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
          "flexDirection": "column-reverse",
        },
        Object {
          "backgroundColor": "#000",
        },
      ]
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        collapsable={undefined}
        pointerEvents="auto"
        style={
          Object {
            "backgroundColor": "#EFEFF4",
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "shadowColor": "black",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 5,
            "top": 0,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
              "height": 1334,
            }
          }
          virtual={undefined}
        >
          <View>
            <View
              style={
                Array [
                  Object {
                    "backgroundColor": "#F8F8F8",
                    "borderBottomColor": "#a7a6ab",
                    "borderBottomWidth": 0.5,
                    "elevation": 3,
                    "flexDirection": "row",
                    "height": 64,
                    "justifyContent": "center",
                    "left": 0,
                    "paddingLeft": 10,
                    "paddingRight": 10,
                    "paddingTop": 15,
                    "right": 0,
                    "shadowColor": undefined,
                    "shadowOffset": undefined,
                    "shadowOpacity": undefined,
                    "shadowRadius": undefined,
                    "top": 0,
                  },
                  Object {
                    "backgroundColor": "gold",
                  },
                ]
              }
              virtual={undefined}
            >
              <View
                style={
                  Object {
                    "alignItems": "flex-start",
                    "alignSelf": "center",
                    "flex": 1,
                  }
                }
                virtual={undefined}
              />
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "alignSelf": "center",
                    "flex": 1,
                  }
                }
                virtual={undefined}
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  numberOfLines={1}
                  style={
                    Object {
                      "color": "#000",
                      "fontFamily": "System",
                      "fontSize": 17,
                      "fontWeight": "600",
                      "textAlign": "center",
                    }
                  }
                  virtual={undefined}
                >
                  Herkyles
                </Text>
              </View>
              <View
                style={
                  Object {
                    "alignItems": "flex-end",
                    "alignSelf": "center",
                    "flex": 1,
                    "flexDirection": "row",
                    "justifyContent": "flex-end",
                  }
                }
                virtual={undefined}
              >
                <View
                  accessibilityComponentType={undefined}
                  accessibilityLabel={undefined}
                  accessibilityTraits={undefined}
                  accessible={true}
                  collapsable={undefined}
                  hasTVPreferredFocus={undefined}
                  hitSlop={undefined}
                  isTVSelectable={true}
                  nativeID={undefined}
                  onLayout={undefined}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "alignSelf": null,
                      "backgroundColor": null,
                      "borderBottomWidth": null,
                      "borderColor": null,
                      "borderLeftWidth": null,
                      "borderRadius": 50,
                      "borderRightWidth": null,
                      "borderTopWidth": null,
                      "elevation": 0,
                      "flexDirection": "row",
                      "height": 45,
                      "justifyContent": "space-between",
                      "marginRight": -8,
                      "opacity": 1,
                      "paddingBottom": 6,
                      "paddingLeft": 15,
                      "paddingRight": 15,
                      "paddingTop": 6,
                      "shadowColor": null,
                      "shadowOffset": null,
                      "shadowOpacity": null,
                      "shadowRadius": null,
                    }
                  }
                  testID={undefined}
                  tvParallaxProperties={undefined}
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                  />
                </View>
              </View>
            </View>
          </View>
          <RCTScrollView
            automaticallyAdjustContentInsets={false}
            contentContainerStyle={undefined}
            contentInset={
              Object {
                "bottom": 0,
              }
            }
            enableAutoAutomaticScroll={true}
            enableResetScrollToCoords={true}
            extraHeight={75}
            extraScrollHeight={0}
            getScrollResponder={[Function]}
            handleOnScroll={[Function]}
            keyboardDismissMode="interactive"
            keyboardOpeningTime={250}
            keyboardShouldPersistTaps="handled"
            keyboardSpace={0}
            onScroll={[Function]}
            resetKeyboardSpace={[Function]}
            resetScrollToCoords={
              Object {
                "x": 0,
                "y": 0,
              }
            }
            scrollEventThrottle={1}
            scrollForExtraHeightOnAndroid={[Function]}
            scrollToEnd={[Function]}
            scrollToFocusedInput={[Function]}
            scrollToPosition={[Function]}
            showsVerticalScrollIndicator={true}
            style={
              Object {
                "backgroundColor": "transparent",
                "flex": 1,
              }
            }
            viewIsInsideTabBar={false}
            virtual={undefined}
          >
            <View>
              <RCTScrollView
                scrollsToTop={true}
              >
                <View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "flex": 1,
                        "justifyContent": "center",
                      }
                    }
                  >
                    <View
                      accessibilityComponentType={undefined}
                      accessibilityLabel={undefined}
                      accessibilityTraits={undefined}
                      accessible={true}
                      collapsable={undefined}
                      hasTVPreferredFocus={undefined}
                      hitSlop={undefined}
                      isTVSelectable={true}
                      nativeID={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function]}
                      onResponderMove={[Function]}
                      onResponderRelease={[Function]}
                      onResponderTerminate={[Function]}
                      onResponderTerminationRequest={[Function]}
                      onStartShouldSetResponder={[Function]}
                      style={
                        Object {
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "center",
                          "marginBottom": 20,
                          "opacity": 1,
                          "padding": 0,
                          "shadowColor": "#303838",
                          "shadowOffset": Object {
                            "height": 5,
                            "width": 0,
                          },
                          "shadowOpacity": 0.35,
                          "shadowRadius": 10,
                        }
                      }
                      testID={undefined}
                      tvParallaxProperties={undefined}
                    >
                      <Image
                        resizeMode="cover"
                        source={1}
                        style={
                          Object {
                            "width": 750,
                          }
                        }
                      />
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 30,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        Campus Recreation & Wellness Center
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 20,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        
Hours
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 15,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        
Weekday: 5AM - 7PM
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 15,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        Weekend: 5:30AM - 8PM
                      </Text>
                    </View>
                  </View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "flex": 1,
                        "justifyContent": "center",
                      }
                    }
                  >
                    <View
                      accessibilityComponentType={undefined}
                      accessibilityLabel={undefined}
                      accessibilityTraits={undefined}
                      accessible={true}
                      collapsable={undefined}
                      hasTVPreferredFocus={undefined}
                      hitSlop={undefined}
                      isTVSelectable={true}
                      nativeID={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function]}
                      onResponderMove={[Function]}
                      onResponderRelease={[Function]}
                      onResponderTerminate={[Function]}
                      onResponderTerminationRequest={[Function]}
                      onStartShouldSetResponder={[Function]}
                      style={
                        Object {
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "center",
                          "marginBottom": 20,
                          "opacity": 1,
                          "padding": 0,
                          "shadowColor": "#303838",
                          "shadowOffset": Object {
                            "height": 5,
                            "width": 0,
                          },
                          "shadowOpacity": 0.35,
                          "shadowRadius": 10,
                        }
                      }
                      testID={undefined}
                      tvParallaxProperties={undefined}
                    >
                      <Image
                        resizeMode="cover"
                        source={1}
                        style={
                          Object {
                            "width": 750,
                          }
                        }
                      />
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 30,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        
Field House
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 20,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        
Hours
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 15,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        
Weekday: 5AM - 7PM
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 15,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        Weekend: 5:30AM - 8PM
                      </Text>
                    </View>
                  </View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "flex": 1,
                        "justifyContent": "center",
                      }
                    }
                  >
                    <View
                      accessibilityComponentType={undefined}
                      accessibilityLabel={undefined}
                      accessibilityTraits={undefined}
                      accessible={true}
                      collapsable={undefined}
                      hasTVPreferredFocus={undefined}
                      hitSlop={undefined}
                      isTVSelectable={true}
                      nativeID={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function]}
                      onResponderMove={[Function]}
                      onResponderRelease={[Function]}
                      onResponderTerminate={[Function]}
                      onResponderTerminationRequest={[Function]}
                      onStartShouldSetResponder={[Function]}
                      style={
                        Object {
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "center",
                          "marginBottom": 20,
                          "opacity": 1,
                          "padding": 0,
                          "shadowColor": "#303838",
                          "shadowOffset": Object {
                            "height": 5,
                            "width": 0,
                          },
                          "shadowOpacity": 0.35,
                          "shadowRadius": 10,
                        }
                      }
                      testID={undefined}
                      tvParallaxProperties={undefined}
                    >
                      <Image
                        resizeMode="cover"
                        source={1}
                        style={
                          Object {
                            "width": 750,
                          }
                        }
                      />
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 30,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        
Fitness East Recreational Facilty
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 20,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        
Hours
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 15,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        
Weekday: 5AM - 7PM
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "fontSize": 15,
                              "textAlign": "center",
                            },
                          ]
                        }
                        uppercase={false}
                        virtual={undefined}
                      >
                        Weekend: 5:30AM - 8PM
                      </Text>
                    </View>
                  </View>
                </View>
              </RCTScrollView>
            </View>
          </RCTScrollView>
        </View>
      </View>
    </View>
  </View>
</View>
`;
